apps/testingaccessibility/src/pages/api/auth/[...nextauth].ts

```diff
@@ -53,6 +53,12 @@ export const nextAuthOptions: NextAuthOptions = {
   ],
   callbacks: {
     async session({session, user, token}) {
+      if (token?.id) {
+        const {getPurchasesForUser} = getSdk()
+        const purchases = await getPurchasesForUser(token.id as string)
```

Comment:
this could potentially be a performance issue but it seemed fast and we were doing it on all the learn routes anyway so this just gives us a very convenient way to access the purchases for a user in middleware etc since they will be on the token